spec.emu

```diff
@@ -15,6 +15,35 @@ markEffects: true
   <p></p>
 </emu-intro>
 
+<emu-clause id="sec-agents">
+  <h1>Agents</h1>
+
+  <emu-table id="table-agent-record" caption="Agent Record Fields">
+    <table>
+      <thead>
+        <tr>
+          <th>Field Name</th>
+          <th>Value</th>
+          <th>Meaning</th>
+        </tr>
+      </thead>
+      <tr>
+        <td>[[UseErrorCaptureStackTraceDataProperty]]</td>
+        <td>a Boolean</td>
```

Comment:
Nit: since this isn't a yes/no, it's an either/or, we would typically use a 2-state enum for this. Something like `[[ErrorCaptureStackTraceStrategy]]` with type "`~data-property~` or `~accessor~`".